### PR TITLE
chore: release toucan-connectors from 4.5.1 to 4.6.0

### PR DESCRIPTION
**Pull request by Sanix-Darker**
## WHAT

This is just a bump from 4.5.1 to 4.6.0 for toucan-connectors.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,13 @@
 
 ## Unreleased
 
+### [4.6.0] 2023-06-02
+
+## Changed
+
+- For a query, if there is missing params, we now either remove it, or raise an Exception in case `handle_errors` is True.
```
Comment (lukapeschke) [inline CHANGELOG.md:9]: This was already the case before... The Exception type has changed though

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,13 @@
...
+- For a given query, we no longer update a pipeline if we found `__VOID__` values in it.
```
Comment (lukapeschke) [inline CHANGELOG.md:10]:
```suggestion
- `__VOID__` values are no longer removed from queries.
```

Comment (Sanix-Darker) [reply CHANGELOG.md]: https://github.com/ToucanToco/toucan-connectors/pull/1107/commits/8e5dc40

File: CHANGELOG.md (hunk 1 above)
Comment (Sanix-Darker) [reply CHANGELOG.md]: ah yes... and i guess it's not necessary for the user to know that info right ?
https://github.com/ToucanToco/toucan-connectors/pull/1107/commits/8e5dc40

Comment (lukapeschke) [reply CHANGELOG.md]: It's necessary to know that the exception has changed, in case it was previously caught

Comment (Sanix-Darker) [reply CHANGELOG.md]: https://github.com/ToucanToco/toucan-connectors/pull/1107/commits/830675377d1b38bf0cc8ad1c8d6d901faa9f8239

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,13 @@
 
 ## Unreleased
 
+### [4.6.0] 2023-06-02
+
+## Changed
+
+- On missing params inside queries with `handle_errors` on **True**, we now raise : `UndefinedVariableError` and not `NonValidVariable`.
```
Comment (lukapeschke) [inline CHANGELOG.md:9]:
```suggestion
- The exception raised by `nosql_apply_parameters_to_query` when `handle_errors` is true and an undefined variable is encountered has changed from  `NonValidVariable`  to `UndefinedVariableError`.
```

* Simple tense, no "we" form
* Function name was missing

**Review by lukapeschke — APPROVED** [review]

**Pull request merged**